lib/Mojo/mysql/Native/Database.pm

```diff
@@ -0,0 +1,227 @@
+package Mojo::mysql::Native::Database;
+use Mojo::Base 'Mojo::mysql::Database';
+
+use Mojo::mysql::Connection;
+use Mojo::mysql::Native::Results;
+use Mojo::mysql::Native::Transaction;
+use Mojo::mysql::Util qw(expand_sql parse_url);
+use Scalar::Util 'weaken';
+use Carp 'croak';
+
+has 'connection';
```

Comment:
Do you think calling this "dbh" will be confusing? It's a handle to the database :smile: